zipline/testing/fixtures.py

```diff
@@ -455,6 +458,41 @@ def pipeline_event_setup_engine(self, dates):
         loader = self.loader_type(*self.pipeline_event_loader_args(dates))
         return SimplePipelineEngine(lambda _: loader, dates, self.asset_finder)
 
+    def get_sids_to_frames(self,
+                           zip_date_index_with_vals,
+                           vals,
+                           date_intervals,
+                           dates):
+        """
+        Construct a DataFrame that maps sid to the expected values for the
+        given dates.
+
+        Parameters
+        ----------
+        zip_date_index_with_vals: callable
+            A function that returns a series of `vals` repeated based on the
+            number of days in the date interval for each val, indexed by the
+            dates in `dates`.
+        vals: iterable
+            An iterable with values that correspond to each interval in
+            `date_intervals`.
+        date_intervals: list
```

Comment:
Per your response to my previous comment, can we update the docs to just note that it's nested one layer deeper?